docs/MVS_parameters.rst

```diff
@@ -1,137 +1,819 @@
+----
 
-**start_date**: The data and time on which the simulation starts at the first step. Acceptable format is YYYY-MM-DD HH:MM:SS. E.g.: 2018-01-01 00:00:00
+.. _developmentcosts-label:
 
-**store_oemof_results**: [Developer setting] Acceptable values are either True or False. Assigning True would enable the results to be stored in a OEMOF file. 
+development_costs
+^^^^^^^^^^^^^^^^^
 
-**timestep**: Length of the timesteps. Acceptable values in minutes. This is currently only tested for 60-minute intervals.
+:Definition: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity.
+:Type: Numeric
+:Unit: currency
+:Example: 10000
+:Restrictions: Positive real numbers
+:Default: None
 
-Common Parameters in the CSV/JSON files and in energyConversion.csv:
---------------------------------------------------------------------
+----
 
-**First row of the csv (C1, E1, D1...)**: Input the names of the conversion components in a computer readable format, ie. with underscores instead of spaces, no special characters (eg. pv_plant_01)
+.. _discountfactor-label:
 
-**age_installed**: The number of years the asset has already been in operation
+discount_factor
+^^^^^^^^^^^^^^^
 
-**development_costs**: A fixed cost to implement the asset, eg. planning costs which do not depend on the (optimized) asset capacity (€)
+:Definition: Discount factor is the factor which accounts for the depreciation in the value of money in the future, compared to the current value of the same money. The common method is to calculate the weighted average cost of capital (WACC) and use it as the discount rate.
+:Type: Numeric
+:Unit: Factor
+:Example: 0.06
+:Restrictions: None
+:Default: None
 
-**specific_costs**: Actual CAPEX of the asset (€/kW), ie. specific investment costs
+----
 
-**efficiency**: Ratio of energy output/energy input
+.. _dispatchprice-label:
 
-**Inflow_direction**: The bus/component from which the energyVector is arriving into the asset
+dispatch_price
+^^^^^^^^^^^^^^
 
-**installedCap**: The already existing installed capacity in-place, which will also be replaced after its lifetime (kW)
+:Definition: Variable cost associated with a flow through/from the asset (€/kWh).
+:Type: Numeric
+:Unit: currency/kWh
+:Example: 0.64
+:Restrictions: In "storage_xx.csv" only the columns "input power" and "output power" require a value, in column "storage capacity" dispatch_price should be set to NaN.
+:Default: None
 
-**maximumCap**: The maximum installable capacity
+----
 
-**label**: Name of the asset
+.. _dsm-label:
 
-**lifetime**: Number of operational years of the asset until it has to be replaced
+dsm
+^^^
 
-**cost_om**: Specific annual OPEX of the asset (€/kW/year)
+:Definition: Stands for Demand Side Management. Currently, not implemented.
+:Type: str
+:Unit: Boolean value
```

Comment:
I find the fields describing a parameter would require a definition and a list of possible values as well. It is hard to understand what is a `:Unit:` in this example:
What is a `Boolean value` as a `:Unit:`. Shouln't `:Type:` be `boolean` and `:Unit:` be None?